server/src/surveyRouteWithConfig.test.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import request from 'supertest';
+import * as SurveyController from './controllers/surveyController';
+import * as SurveyDBHandler from './databaseHandlers/surveyDBHandler';
+
+jest.mock('./utils/getConfig', () => {
+  return {
+    getServerConfig: () => ({
+      communicationServicesConnectionString: 'endpoint=your_endpoint;accesskey=secret',
+      microsoftBookingsUrl: 'https://testBookingsUrl',
+      chatEnabled: true,
+      screenShareEnabled: true,
+      companyName: 'Lamna Healthcare',
+      colorPalette: '#0078d4',
+      waitingTitle: 'Thank you for choosing Lamna Healthcare',
+      waitingSubtitle: 'Your clinician is joining the meeting',
+      logoUrl: '',
+      postCall: {
+        survey: {
+          type: 'onequestionpoll',
+          options: {
+            prompt: 'testing prompt',
+            pollType: 'likeOrDislike',
+            saveButtonText: 'like or dislike'
+          }
+        }
+      },
+      cosmosDb: {
+        endpoint: 'http://testinghost.com',
+        dbName: 'testingDBName'
+      }
+    })
+  };
+});
+
+describe('Tes survey route', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('check if /api/surveyResults route is open with cosmosDb configs', async () => {
+    const mockedSurveyDBHandler = {
+      init: jest.fn()
+    };
+    const mockStoreSurveyResult = jest
+      .spyOn(SurveyController, 'storeSurveyResult')
+      .mockImplementationOnce(() => async () => Promise.resolve());
+
+    jest.spyOn(SurveyDBHandler, 'createSurveyDBHandler').mockReturnValueOnce(mockedSurveyDBHandler as any);
+
+    (await import('./app')).default;
```

Comment:
I see that this approach works when moved to `app.test.ts`. Should we move these tests too so that all test cases are in one place?

Reply:
hmm I will try. The reason I was able to merge it because it didn't require the `postCall` and `cosmosDb` configs. I will try to merge it.

Reply:
Added test in `app.ts` to testing if the route is close when there is no config for cosmosDb.